frontend/src/components/Explanation.js

```diff
@@ -1 +1,17 @@
-// This is where the explanation component will be created
\ No newline at end of file
+import React from 'react';
+import './Explanation.css';
+function Explanation() {
+    return (
+        <div class="jumbotron jumbotron-fluid">
+            <div class="container">
+                <h1 class="display-4">What is this?</h1>
+                <p class="lead">This is a project powered by students that gives you the power to translate any news article to your preferred langauge :).</p>
+            </div>
+            <div>
+                <a class="btn btn-primary btn-lg button" href="#Explanation.css" role="button" >Learn more</a>
```

Comment:
We can actually remove this button completely as it doesnt do anything. Doing so also removes the needed Explanation .css